Solutions/Cynerio/Data/Solution_Cynerio.json

```diff
@@ -0,0 +1,25 @@
+{
+    "Name": "Cynerio",
+    "Author": "micha@cynerio.co",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Cynerio.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "The [Cynerio](https://www.cynerio.com/) solution for Microsoft Sentinel enables you to ingest Cynerio Security Events to the Microsoft Sentinel platform, providing more insight into your organization network security posture and improving your security operation capabilities.",
+    "Data Connectors": [
+        "Solutions/Cynerio/Data Connectors/Cynerio_Connector.json"
+    ],
+    "Parsers": [
+        "Solution/Cynerio/Parsers/CynerioEvent_Authentication.txt",
+        "Solution/Cynerio/Parsers/CynerioEvent_NetworkSession.txt"
```

Comment:
The file path for parser is incorrect,
It should be ( Solutions ) rather than "Solution".
Please repackage the solution the script will work after this change.
Thank you.

Reply:
done, thank you